### DISCUSSION TITLE
ScriptReference/Mathf.Sign

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#503] [Notes]
# ScriptReference/Mathf.Sign

https://docs.unity3d.com/ScriptReference/Mathf.Sign.html

**Comment by Raveler** [comment]
Probably the most idiotic decision in the entire Unity API to have Mathf.Sign(0) not return 0. Makes this method pretty much useless in a lot of cases.
